rules/linux/file_create/file_create_lnx_triple_cross_rootkit_persistence.yml

```diff
@@ -0,0 +1,23 @@
+title: Triple Cross eBPF Rootkit Default Persistence
+id: 1a2ea919-d11d-4d1e-8535-06cda13be20f
+status: experimental
+description: Detects the creation of "ebpfbackdoor" files in both "cron.d" and "sudoers.d" directories. Which both are related to the TripleCross persistence method
+references:
+    - https://github.com/h3xduck/TripleCross/blob/master/apps/deployer.sh
+author: Nasreddine Bencherchali
+date: 2022/07/05
+logsource:
+    product: linux
+    category: file_create
+detection:
+    selection:
+        TargetFilename|endswith: 'ebpfbackdoor'
```

Comment:
Yes, I've seen that one but since it's a generic one and medium. I thought a specific one for this one in high is a good idea? If you think it's not worth it I'll remove it.

Reply:
I'll also create a generic one for "sudoers.d"

Reply:
I've renamed both rules to indicate they are looking for persistence. Check and let me know

Reply:
Thanks , now we have 2 medium "generic" and 1 high specific